snp_prbt_bringup/launch/hardware_bringup.launch

```diff
@@ -33,9 +36,9 @@
   <node name="snp_prbt_support" pkg="snp_prbt_support" type="spindle_sim" if="$(arg sim_robot)"/>
 
   <!-- camera -->
-  <include file="$(find snp_prbt_support)/launch/setup_camera.launch" unless="$(arg sim_robot)">
+  <!-- <include file="$(find snp_prbt_support)/launch/setup_camera.launch" unless="$(arg sim_robot)">
```

Comment:
Can we delete this completely maybe? We will always be launching the camera separately